### PR TITLE
destination-google-sheets: [autopull] base image + poetry + up_to_date

### PR DESCRIPTION
**Pull request by natikgadzhi**

### CODE REVIEW

File: airbyte-integrations/connectors/destination-google-sheets/pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+[build-system]
+requires = [ "poetry-core>=1.0.0",]
+build-backend = "poetry.core.masonry.api"
+
+[tool.poetry]
+version = "0.2.4"
```
Comment (natikgadzhi) [inline airbyte-integrations/connectors/destination-google-sheets/pyproject.toml:6]:
```suggestion
version = "0.2.5"
```

File: airbyte-integrations/connectors/destination-google-sheets/README.md [hunk 2]
```diff
@@ -1,116 +1,91 @@
-# Google Sheets Destination
+# Google-Sheets source connector
```
Comment (lazebnyi) [inline airbyte-integrations/connectors/destination-google-sheets/README.md:1]:
```suggestion
# Google-Sheets destination connector
```

File: airbyte-integrations/connectors/destination-google-sheets/README.md [hunk 3]
```diff
@@ -1,116 +1,91 @@
...
-### Publishing a new version of the connector
+Please commit the changes to `pyproject.toml` and `poetry.lock` files.
 
+## Publishing a new version of the connector
 You've checked out the repo, implemented a million dollar feature, and you're ready to share your changes with the world. Now what?
-
 1. Make sure your changes are passing our test suite: `airbyte-ci connectors --name=destination-google-sheets test`
-2. Bump the connector version in `metadata.yaml`: increment the `dockerImageTag` value. Please follow [semantic versioning for connectors](https://docs.airbyte.com/contributing-to-airbyte/resources/pull-requests-handbook/#semantic-versioning-for-connectors).
+2. Bump the connector version (please follow [semantic versioning for connectors](https://docs.airbyte.com/contributing-to-airbyte/resources/pull-requests-handbook/#semantic-versioning-for-connectors)): 
+    - bump the `dockerImageTag` value in in `metadata.yaml`
```
Comment (lazebnyi) [inline airbyte-integrations/connectors/destination-google-sheets/README.md:84]:
```suggestion
    - bump the `dockerImageTag` value in `metadata.yaml`
```

File: airbyte-integrations/connectors/destination-google-sheets/README.md [hunk 4]
```diff
@@ -1,116 +1,91 @@
-# Google Sheets Destination
+# Google-Sheets source connector
 
-This is the repository for the Google Sheets destination connector, written in Python.
-For information about how to use this connector within Airbyte, see [the documentation](https://docs.airbyte.io/integrations/destinations/google-sheets).
+
+This is the repository for the Google-Sheets source connector, written in Python.
```
Comment (lazebnyi) [inline airbyte-integrations/connectors/destination-google-sheets/README.md:4]:
```suggestion
This is the repository for the Google-Sheets destination connector, written in Python.
```

**Review by lazebnyi — APPROVED** [review]
lgtm, just minor updates required

**Pull request merged**